softgroup/services/profile/profile-impl/src/test/java/com/softgroup/profile/factory/ProfileCommandHandlerFactoryTest.java

```diff
@@ -0,0 +1,145 @@
+package com.softgroup.profile.factory;
+
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.RequestBuilder;
+import com.softgroup.profile.api.router.ProfileRequestHandler;
+import com.softgroup.profile.impl.factory.ProfileCommandHandlerFactory;
+import com.softgroup.profile.impl.handler.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProfileCommandHandlerFactoryTest {
+
+    private static final String PROFILE_TYPE_NAME = "profile";
+
+    private static final String CMD_CONTACTS_SYNC_NAME        = "contacts_sync";
+    private static final String CMD_GET_CONTACT_PROFILES_NAME = "get_contact_profiles";
+    private static final String CMD_GET_MY_PROFILE_NAME       = "get_my_profile";
+    private static final String CMD_SET_MY_PROFILE_NAME       = "set_my_profile";
+    private static final String CMD_GET_OTHER_PROFILES_NAME   = "get_other_profiles";
+    private static final String CMD_GET_PROFILE_SETTINGS_NAME = "get_profile_settings";
+    private static final String CMD_SET_PROFILE_SETTINGS_NAME = "set_profile_settings";
+    private static final String CMD_GET_LAST_TIME_ONLINE      = "get_last_time_online";
+
+    @Spy
+    private List<ProfileRequestHandler> listHandlers = new ArrayList<>();
+
+    @InjectMocks
+    private ProfileCommandHandlerFactory factory;
+
+    private ProfileRequestHandler cmdContactsSyncHandler       = new ProfCmdContactsSyncHandler();
```

Comment:
@VasjaVn Look at your test. It's almost  impossible to understand what it's testing.